test/test-eth-ledger-bridge-keyring.js

```diff
@@ -385,19 +398,6 @@ describe('LedgerBridgeKeyring', function () {
     })
   })
 
-  describe('signMessage', function () {
-    it('should call create a listener waiting for the iframe response', function (done) {
-
-      chai.spy.on(window, 'addEventListener')
-      setTimeout((_) => {
-        keyring.signPersonalMessage(fakeAccounts[0], '0x123')
```

Comment:
:sweat_smile:  Well now.

Technically there is a `signMessage` public method that isn't being tested though. It's a bit silly, in that it's just an alias of `signPersonalMessage`. But, maybe worth duplicating the `signPersonalMessage` test for `signMessage` as well, to ensure the alias is tested and works the same way?